src/base/bittorrent/session.cpp

```diff
@@ -321,6 +321,11 @@ QString Session::tempPath() const
     return m_tempPath;
 }
 
+QString Session::torrentTempPath(const InfoHash &hash) const
+{
+    return m_tempPath + QString(hash) + "/";
```

Comment:
Content can be either legal or illegal, regardless of transport.
If I have some linux distro, it's legal. But if I have, say, some unlicensed software distro, it's is illegal, regardless of whether I downloaded it via BitTorrent or bought the "pirate" disc with it.